{
  body_to_camera_quat: [0.5,0.5,0.5,0.5],

  body_to_camera_trans: [0.1354765750332004, 0.0175314092839309, -0.0455417178618858]
}
